CLDR-18940 Further updates to release page (#5054)
unicode-org/cldr: docs/site/downloads/cldr-48.md

       - Kashmiri
       - Kurdish (Kurmanji, Latin)
       - Kurdish (Central, Arabic)
       - Kurdish (Southern, Arabic)
       - Kyrgyz
- Many features selectable with locale options now have `scope="core"` names, for better presentation in menus
   - Calendar names, collation names, emoji options, currency formats, hour-cycle options, and so on.
   - Rather than seeing
       - Calendar
           - Buddhist Calendar
           - Chinese Calendar
           - Gregorian Calendar
    - Users can see
        - Calendar
           - Buddhist
           - Chinese
           - Gregorian
- Recent or upcoming currency names were added (XCG, ZWG)
- To match ISO, translations for the region Sark (CQ) was added.
- There are now combination formats for relative times.
    - A combining pattern is used to combine a date with a time, such as “March 20 _at_ 12:30”.
In some languages the use of a relative date such as “tomorrow” or “2 days ago” requires a _different_ combining pattern than for a fixed date like “March 20”.
A new “relative” variant is introduced to allow for those languages.

> - Some additional flexible (aka available) date formats were added.

Author: aka means something slightly different.

> - Some additional flexible (aka available) date formats were added.

Reviewer: I thought a.k.a. was short for "also known as"?

- Many locales had seldom-used short timezone abbreviations (such as EST) removed, or moved to sublocales that use them.
- The currency-number formats for alphaNextToNumber, noCurrency, and compact currency formats are now generated from other data for consistency.

> The alphaNextToNumber patterns allow for a space between letter currency symbols and numbers, so "USD 123" vs "$123".

Author: should be 'eg,"

## Data Changes

### Locale Changes

#### General
- Languages that reached Basic in the last release have their names translated at Modern Coverage in this release.
- Compound language names now have "core" and "extension" variants for more uniform formats in menus and lists
   - For example, that allows the Kurdish variants to have a uniform format where more than Kurmanji is displayed.
       - Kashmiri
       - Kurdish (Kurmanji, Latin)
       - Kurdish (Central, Arabic)
       - Kurdish (Southern, Arabic)
       - Kyrgyz
- Many features selectable with locale options now have `scope="core"` names, for better presentation in menus
   - Calendar names, collation names, emoji options, currency formats, hour-cycle options, and so on.
   - Rather than seeing
       - Calendar
           - Buddhist Calendar
           - Chinese Calendar
           - Gregorian Calendar
    - Users can see
        - Calendar
           - Buddhist
           - Chinese
           - Gregorian
- Recent or upcoming currency names were added (XCG, ZWG)
- To match ISO, translations for the region Sark (CQ) was added.

> - There are now combination formats for _relative_ dates + times, such as “March 20 _at_ 12:30”.

Reviewer 2: ```suggestion
- There are now combination formats for _relative_ dates + times, such as “tomorrow _at_ 12:30”.
```

Reviewer 2: Updating to match the other relative time fix.

In some languages the use of a relative date such as “tomorrow” or “2 days ago” requires a _different_ combining pattern than for a fixed date like “March 20”.
A new “relative” variant is introduced to allow for those languages.

> - Some additional flexible (aka available) date formats were added.

Author: I'll try a change for that.
```suggestion
- Some additional flexible date formats were added. (aka `availableFormats`)
```

- Many locales had seldom-used short timezone abbreviations (such as EST) removed, or moved to sublocales that use them.
- The currency-number formats for `alphaNextToNumber`, `noCurrency`, and compact currency formats are now generated from other data for consistency.

> The alphaNextToNumber patterns allow for a space between letter currency symbols and numbers, so "USD 123" vs "$123".

Author: ```suggestion
The alphaNextToNumber patterns allow for a space between letter currency symbols and numbers. For example, "USD 123" vs "$123".
```

- The tooling made it easier to see when a space was a non-breaking character or not, or thin versions of those. The usage is now more consisent in many locales.
- New emoji for Unicode 17 have added names and search keywords.
- For the Etc/Unknown timezone, the exemplarCity name was changed from “Unknown City” to “Unknown Location”, for clarity.
- Rational number formats were added, allowing for formats like 5½.
- Certain concentration units were reworked, for “parts per million”, “parts per billion”

> - Additional units were added for scientific contexts (coulombs, farads, teslas, etc.) and for English systems (fortnights, imperial pints, etc.). However, the English system names were not required for translation.

Reviewer 2: translation of those names were not in coverage. I don't think we've checked whether people added them at comprehensive

## Migration

- Number patterns that did not have a specific numberSystem (such as latn or arab) had be deprecated for many releases, and were finally removed.

> - The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week is will be more clearly targeted at matching usage in displayed month calendars.

Author: ```suggestion
```

- Additionally, language and territory data in `languageData` and `territoryInfo` data received significant updates to improve accuracy and maintainability [CLDR-18087]
- The likely language for Belarus changed to Russian [CLDR-14479]

> - The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 

Reviewer 2: This needs to move to be a CLDR 49 advanced warning as we have not removed the eras yet https://unicode-org.atlassian.net/browse/CLDR-11400

> - The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 
> and feedback that the general practice for exact dates is to use Gregorian for pre-Meiji dates.

Author: ```suggestion
```

- **TBD Additional items plus future guidance will be added before the beta, on Oct 1.**

### V49 advance warnings

> The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.

Author: ```suggestion
The following changes are planned for CLDR 49. Please plan accordingly to avoid disruption.
- The default week numbering changes to ISO instead being based on the calendar week starting in CLDR 48 [CLDR-18275]. The calendar week is will be more clearly targeted at matching usage in displayed month calendars.
- The pre-Meiji Japanese eras were removed: There was too much uncertainty in the exact values them 
and feedback that the general practice for exact dates is to use Gregorian for pre-Meiji dates.
```
